bo/marktteilnehmer.go

```diff
@@ -8,7 +8,7 @@ import (
 // Marktteilnehmer models participants of the German energy market
 type Marktteilnehmer struct {
 	Geschaeftspartner
-	Marktrolle       marktrolle.Marktrolle       `json:"marktrolle,omitempty" validate:"required"`                             // Marktrolle gibt im Klartext die Bezeichnung der Marktrolle an. Details siehe ENUM Marktrolle
+	Marktrolle       *marktrolle.Marktrolle      `json:"marktrolle,omitempty" validate:"required"`                             // Marktrolle gibt im Klartext die Bezeichnung der Marktrolle an. Details siehe ENUM Marktrolle
```

Comment:
if `marktrolle` should be nullable, do we not need to remove the `validate:"required"`  ?

Reply:
Hmmmm, probably. It's not required in the .NET package. I changed it.